src/SFA.DAS.EmployerApprenticeshipsService.Domain/Interfaces/IDasLevyService.cs

```diff
@@ -20,5 +20,7 @@ public interface IDasLevyService
         Task<IEnumerable<DasEnglishFraction>> GetEnglishFractionHistory(long accountId, string empRef);
 
         Task<int> GetPreviousAccountTransaction(long AccountId, DateTime FromDate);
+
+        Task<string> GetProviderName(int ukprn, long accountId, string periodEnd);
```

Comment:
This is a duplication of logic because the API and web controllers don't use the same logic. This is a copy and paste of the IDasLevyService method from the Accounts domain.